donate/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
Could this file be moved to the root? This removes the need for ``donate/donate``

Reply:
Yep, will do.